### ISSUE TITLE
The Third Age(TM) [The Lord of the Rings: The Third Age] [4541005F]

### ISSUE THREAD

**Issue by ergo720** [Cxbx-Reloaded/game-compatibility#386]
### Description

Unsupported, LTCG

```
---------------------------
Cxbx-Reloaded
---------------------------
[0x1FDC] EmuMain: Received Fatal Message:

* LTCG Title Detected: This game is not supported by HLE

---------------------------
OK   
---------------------------

```

### Dumps 

[Xbe.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/1327472/Xbe.txt)

### Status
**Previous behavior:** 

None reported

**Current behavior:**

Nothing

**Comment by CakeLancelot** [comment]
Is this a multi-xbe title? According to the xbe dump the title is just "LOTR3".

**Comment by ergo720** [comment]
No, it's a single xbe. LOTR3 is probably some kind of acronym (Lord Of Third?).

**Comment by CakeLancelot** [comment]
Okay well the xbe title should always be first with a more detailed title following in brackets - I'll go ahead and correct it.

**Comment by ergo720** [comment]
LLE_GPU: cxbx just quits the emulation after a while, with no errors and no video/sound at all. From the log I see that EmuX86_Read EmuX86_Write are being repeatedly called on unknown addresses, an by the looks of it there seems to be a nullptr floating around
https://github.com/Cxbx-Reloaded/Cxbx-Reloaded/commit/6630ee4c590e085635764a7e66e4cd360508d2f1
[KrnlDebug.zip](https://github.com/Cxbx-Reloaded/game-compatibility/files/1866779/KrnlDebug.zip)

**Comment by gandalfthewhite19890404** [comment]
Latest dev-build, with LLE GPU goes to dirty disk error after FMVs, need to turn on all-cores-hack, without it it stucks after boot. FMV works perfect.
